app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarCFRPresenter.kt

```diff
@@ -118,8 +118,7 @@ class BrowserToolbarCFRPresenter(
                 color = FirefoxTheme.colors.textOnColorPrimary,
                 modifier = Modifier.clickable {
                     context.components.useCases.tabsUseCases.selectOrAddTab.invoke(
-                        SupportUtils.getSumoURLForTopic(
-                            context,
+                        SupportUtils.getGenericSumoURLForTopic(
```

Comment:
`getSumoURLForTopic` was not building the correct URL, therefore `getGenericSumoURLForTopic` is used instead.